app/views/hosts/console/vmrc.html.erb

```diff
@@ -0,0 +1,15 @@
+<% title "#{@console[:name]}" %>
+<div class="blank-slate-pf">
+  <div class="blank-slate-pf-icon">
+    <%= icon_text('screen', '', :kind => 'pficon') %>
+  </div>
+  <h1><%= _('VMRC Console') %></h1>
+  <p><%= _('To launch the VMRC console, you need the VMRC software installed.') %></p>
```

Comment:
I assume that there is no way for us to detect it? (if its installed or not?)

Reply:
Yes, as far as I know, we can't detect if it's installed or not.